sdk/tools/mkisofs/schilytools/libmdigest/byte_order.h

```diff
@@ -61,7 +61,7 @@ extern "C" {
 		__BYTE_ORDER == __LITTLE_ENDIAN) || \
 	defined(CPU_IA32) || defined(CPU_X64) || \
 	defined(__ia64) || defined(__ia64__) || defined(__alpha__) || defined(_M_ALPHA) || \
-	defined(vax) || defined(MIPSEL) || defined(_ARM_)
+	defined(vax) || defined(MIPSEL) || defined(_ARM_) || (defined(__arm64__) && defined(__APPLE__))
```

Comment:
Same here, but maybe also it's worth to check whether `_M_ARM64` or `_ARM64_` are defined here too:
```suggestion
	defined(vax) || defined(MIPSEL) || defined(_ARM_) || (defined(__arm64__)
```